### PR TITLE
Update web installer

### PR DESCRIPTION
**Pull request by Boy132**
- Removed user check in installer
- Added back old .env variable for installer
- Simplified `p:environment:setup` command even more
- Fixes auto login
- The `APP_URL` is now auto filled
- The setup work is now done after each step (e.g. writing to .env, running migrations, creating the admin user) rather than at the very end
- Removed "unsaved changes" alert
- Improved error handling and reporting
- New "Setup complete" step that handles the queue and crontab stuff:
grafik
- Changed `.env.example` to set a `SESSION_COOKIE`. This way the page doesn't expire when changing the app name. [(By default, Laravel uses the `APP_NAME` to generate the name of the cookie)](https://github.com/laravel/framework/blob/11.x/config/session.php#L130-L133)

Supersedes #595

### CODE REVIEW

File: app/Filament/Pages/Installer/PanelInstaller.php [hunk 1]
```diff
@@ -32,54 +34,43 @@ class PanelInstaller extends SimplePage implements HasForms
...
     {
         return MaxWidth::SevenExtraLarge;
     }
 
-    public static function show(): bool
+    public static function isInstalled(): bool
     {
-        if (User::count() <= 0) {
-            return true;
-        }
```
Comment (lancepioch) [inline app/Filament/Pages/Installer/PanelInstaller.php:51]: I definitely need this in there, it's a much better user experience.

File: app/Filament/Pages/Installer/PanelInstaller.php [hunk 2]
```diff
@@ -32,54 +34,43 @@ class PanelInstaller extends SimplePage implements HasForms
...
-
-        if (config('panel.client_features.installer.enabled')) {
-            return true;
-        }
-
-        return false;
+        // This defaults to true so existing panels count as "installed"
+        return env('APP_INSTALLED', true);
```
Comment (lancepioch) [inline app/Filament/Pages/Installer/PanelInstaller.php:53]: Should be using config instead of env.

File: .env.example [hunk 3]
```diff
@@ -4,6 +4,7 @@ APP_KEY=
 APP_TIMEZONE=UTC
 APP_URL=http://panel.test
 APP_LOCALE=en
+APP_INSTALLED=false
```
Comment (lancepioch) [inline .env.example:7]: I really don't like this being necessary for a default app.

Anybody who copies this .env over will automatically have the installer enabled which might be a problem.

**Review by lancepioch — CHANGES_REQUESTED** [review]

File: app/Filament/Pages/Installer/PanelInstaller.php (hunk 1 above)
Comment (notAreYouScared) [reply app/Filament/Pages/Installer/PanelInstaller.php:51]: I disagree. This ONLY applies to docker... You're also forcing docker users to use SQLite in the beginning.

**Review by notAreYouScared — APPROVED** [review]

File: .env.example (hunk 3 above)
Comment (notAreYouScared) [reply .env.example:7]: We can revisit this. For now i'm going to merge it so the installer isn't broken for every new install.

File: app/Filament/Pages/Installer/PanelInstaller.php (hunk 2 above)
Comment (notAreYouScared) [reply app/Filament/Pages/Installer/PanelInstaller.php:54]: We can revisit this. For now i'm going to merge it so the installer isn't broken for every new install.

**Pull request merged**